src/racing/actors.c

```diff
@@ -402,88 +406,43 @@ void init_red_shell_texture(void) {
         blue_color = color_pixel & 0x3E;
         alpha_color = color_pixel & 0x1;
 
-        *red_shell_texture = (red_color >> 5) | (green_color << 5) | blue_color | alpha_color; // Invert color
+        *red_shell_texture = (red_color >> 5) | (green_color << 5) | blue_color | alpha_color; // Invert green and red
```

Comment:
// Invert X to Y would be better
like

// Invert green to red

But with that wording, invert might not work anymore maybe
// Transform green to red